packages/styles/tokens/colors.scss

```diff
@@ -102,22 +91,4 @@ $colors: (
   hig-red-alert: rgba(236, 74, 65, 1),
 );
 
-// Dynamically create css classes that can be used with divs
-// This example will add "hig-blue-30" as background color to this div:
-// <div class='hig__colors__hig-blue-30--background'></div>
-
-@each $color_name, $color_value in $colors {
-  :export {
-    #{$color_name}: $color_value;
-  }
-
-  .hig__colors__#{$color_name}--color {
-    color: $color_value;
-  }
-  .hig__colors__#{$color_name}--background {
-    background-color: $color_value;
-  }
-  .hig__colors__#{$color_name}--border {
-    border-color: $color_value;
-  }
-}
+$typography-base-color: map-get($colors, 'hig-gray-70');
```

Comment:
so this totally makes sense. but we use  this `color` function all over the code base not just in typography to retrieve colors.  Has this been updated? Maybe I am not seeing it.

Reply:
We split the colors and spacing files in two. `tokens/colors` contains raw values for colors, and `mixins/colors` contains the SASS function to fetch the color by name. The intention is that tokens can potentially be reused for other platforms (Desktop, iOS), or even other CSS-based tools (SASS, PostCSS, LESS). Yes, this particular color map happens to use a SASS feature, but today's separation should make a future refactor easier, if it's needed.

Using `map-get` here was a shortcut to avoid any imports in token files.

Reply:
oh ok. that makes sense

Reply:
oh I see it now in the file above.